proto/google/api/http.proto

```diff
@@ -0,0 +1,370 @@
+// Copyright 2025 Google LLC
```

Comment:
This is from the instructions in:  https://github.com/grpc-ecosystem/grpc-gateway?tab=readme-ov-file#2-with-custom-annotations